pkg/services/pluginsintegration/plugincontext/plugincontext.go

```diff
@@ -62,37 +82,58 @@ func (p *Provider) Get(ctx context.Context, pluginID string, user identity.Reque
 		pCtx.AppInstanceSettings = appSettings
 	}
 
+	ua, err := useragent.New(p.cfg.BuildVersion, runtime.GOOS, runtime.GOARCH)
+	if err != nil {
+		p.logger.Warn("Could not parse user agent", "error", err)
```

Comment:
Nit: I think it would be clearer like this:
```suggestion
		p.logger.Warn("Could not create user agent", "error", err)
```